File: python/udemy/6-vetores/negativos.py
# Faça um programa que leia um número inteiro positivo N (máximo = 10) e depois N números inteiros e armazene-os em um vetor. Em seguida, mostrar na tela todos os números negativos lidos.

n = int(input('Número inteiro positivo até 10: '))
vet = []

for i in range(n):
    num = int(input('Número inteiro: '))
    vet.append(num)

for x in vet:
    if x < 0:
        print(x)

File: c/udemy/6-vetores/negativos.c
// Faça um programa que leia um número inteiro positivo N (máximo = 10) e depois N números inteiros e armazene-os em um vetor. Em seguida, mostrar na tela todos os números negativos lidos

#include <stdio.h>

int main() {

    int n;

    printf("Quantos valores? ");
    scanf("%d", &n);

    int vet[n];

    for (int i=0; i<n; i++) {
        printf("Valor: ");
        scanf("%d", &vet[i]);
    }

    printf("Valores negativos:\n");

    for (int i=0; i<n; i++) {
        if (vet[i] < 0) {
            printf("%d\n", vet[i]);
        }
    }

    return 0;

}